deploy/kubernetes/clusterrole.yaml

```diff
@@ -78,6 +80,7 @@ rules:
   resources:
   - policies
   verbs:
+  - create
```

Comment:
I added these three new "create" perms to my istio branch that I will use when I submit our PR for upgrading istio helm charts when we release kiali v0.14. See https://github.com/istio/istio/compare/master...jmazzitelli:kiali-next-version-changes?expand=1